tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -431,7 +433,7 @@ public:
       return fSampleNameToEventEntries.size();
    }
 };
-
+} // namespace Experimental
```

Comment:
Could you put the changing of namespace of the `ProgressHelper` class in a separate commit? And thanks for noticing it, this slipped in my reviews of the previous PR